utils/scarb-test-support/src/simple_http_server.rs

```diff
@@ -104,6 +140,22 @@ async fn logger<B>(
         );
     }
 
+    let res_status = response.status();
+    let res_headers = response.headers().clone();
+
+    let log = HttpLog {
+        req_method,
+        req_uri,
+        req_headers,
+        res_status,
+        res_headers,
+    };
+
+    {
+        let mut logs = logs.lock().await;
```

Comment:
Would it make sense, to have some flag that disables logging entirely, so we can avoid the performance hit? (I know this server is primarily a test helper, but might be useful for some quick demo / test 🤔 )

Reply:
I doubt this will be a performance issue ever, there are more serious things going on here, like buffering entire response in the middleware. Let's avoid optimizing this part, it's really just meant for very low traffic tests, and it doesn't make sense to add more code here.

If you want a demo, you'd better be with `python3 -m http.server` or Caddy 😉